### DISCUSSION TITLE
Accessing AWS from an iOS application

### DISCUSSION THREAD

**Discussion by rpatel8del** [awslabs/aws-sdk-swift#724] [Q&A]
Hi there,

I'm trying to call/implement the `SetUserMFAPreference` from an iOS app, Amplify doesn't support this natively so I'm using the `escapeHatch`. I've gone through the doc, I've setup my AWS access credentials in my env var of xcode and this works fine in development. 

Whats the best practice for baking in the access credentials in the app to the SDK can access it in production?

Cheers

**Comment by lon9man** [comment]
2 years are passed. any reaction?
